### ISSUE TITLE
Can't resolve '@fortawesome/free-regular-svg-icons'

### ISSUE THREAD

**Issue by fearless23** [GetStream/react-activity-feed#271]
## The issue occurs on vercel build, but no errors in local build
```txt
ModuleNotFoundError: Module not found: Error: Can't resolve '@fortawesome/free-regular-svg-icons' in '/vercel/path0/node_modules/react-activity-feed/dist/components'

12:47:13.325 | Error: &gt; Build failed because of webpack errors
12:47:13.325 | at /vercel/path0/node_modules/next/dist/build/index.js:17:924
12:47:13.325 | at runMicrotasks (&lt;anonymous&gt;)
12:47:13.327 | at processTicksAndRejections (internal/process/task_queues.js:97:5)
12:47:13.327 | at async Span.traceAsyncFn (/vercel/path0/node_modules/next/dist/telemetry/trace/trace.js:6:584)
```

**Comment by fearless23** [comment]
React: 16.13.0
Node: 12.21.0
Next JS: 10.12.0

Tested with multiple `react-activity-feed` versions:
- 1.0.0
- 1.1.0
- 1.1.1

**Comment by fearless23** [comment]
Fixed by installing
- "@fortawesome/free-regular-svg-icons": "^5.15.3" 
- "@fortawesome/react-fontawesome": "^0.1.14"

Above is a hack and these packages should be part of `react-activity-feed`

**Comment by fearless23** [comment]
I see above packages used in 
### `src/components/Audio.tsx` 
```tsx
import React, { useRef, useState, useEffect, MouseEventHandler } from 'react';
import { IconButton } from 'react-file-utils';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import { faPlayCircle, faPauseCircle } from '@fortawesome/free-regular-svg-icons';
import { OGAPIResponse } from 'getstream';

... rest of the code
```
and 
### `src/components/StatusUpdateForm.tsx`
```tsx
import React, { ReactNode } from 'react';
import { Activity, NewActivity, UR } from 'getstream';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import { faBookmark } from '@fortawesome/free-regular-svg-icons';

...rest of the code
```

**Comment by arnautov-anton** [comment]
Thank you for bringing this up! Code with these packages should've been replaced with custom icons due to final bundle size. Will take a look ASAP.

**Comment by arnautov-anton** [comment]
Hey, @fearless23! Fixed the issue in #272, closing for now! :)
